### PR TITLE
Bridge

### PR DESCRIPTION
**Pull request by mmiedzian**
Maciej Miedzianowski
195016
Grupa: IO2

### CODE REVIEW

File: src/main/java/edu/kis/vh/stacks/implementations/StackArray.java [hunk 1]
```diff
@@ -5,7 +5,7 @@
  */
 public class StackArray implements StackInterface {
 
-    private final int STACK_SIZE = 12;
+    private final int STACK_SIZE = 15;
```
Comment (kowallus) [inline src/main/java/edu/kis/vh/stacks/implementations/StackArray.java]: Why?

File: src/main/java/edu/kis/vh/stacks/implementations/StackArray.java [hunk 2]
```diff
@@ -5,7 +5,7 @@
...
     private int[] ITEMS = new int[STACK_SIZE];
```
Comment (kowallus) [inline src/main/java/edu/kis/vh/stacks/implementations/StackArray.java]: items is not a constant

File: src/main/java/edu/kis/vh/stacks/implementations/StackInterface.java [hunk 3]
```diff
@@ -5,7 +5,7 @@
  */
 public interface StackInterface {
 
-    int EMPTY_STACK = -1;
+    int EMPTY_STACK = 0;
```
Comment (kowallus) [inline src/main/java/edu/kis/vh/stacks/implementations/StackInterface.java]: this should be: public final int

Comment (mmiedzian) [inline src/main/java/edu/kis/vh/stacks/implementations/StackInterface.java]: All fields in interface implicitly are "public static final"

Comment (kowallus) [inline src/main/java/edu/kis/vh/stacks/implementations/StackInterface.java]: OK +0.5pts. Reasonable :). I think I prefer explicit declarations.
Total: 2.5 pts

**Pull request closed without merging**